packages/passport-crypto/test/crypto.spec.ts

```diff
@@ -22,7 +22,8 @@ describe("Passport encryption", function () {
       encryptionKey
     );
     const decrypted = await passportDecrypt(encrypted, encryptionKey);
-    const parsed = JSON.parse(decrypted);
+    const parsed =
```

Comment:
i'm not sure if this is a flakey test - but it failed locally until I did this

Reply:
this test has never flaked in my experience